### ISSUE TITLE
The table formatter does not respect alignment!

### ISSUE THREAD

**Issue by harshcurious** [roman-r-m/joplin-plugin-table-formatter#6]
The table formatter does not respect alignment specified.

Just try and align the following table:
```
| A   | B   | C   | D   |
| --- | ---: | :---: | :--- |
| dhdhgfgh | dsgsdgdg | dsgsdfg | zxczc |
```
You will get the following (which is not a table according to markdown syntax):
```
| A        | B        | C       | D     |
|----------|----:-----|-:---:---|-:-----|
| dhdhgfgh | dsgsdgdg | dsgsdfg | zxczc |
```

I should have got:
```
| A        | B        | C       | D     |
|----------|---------:|:-------:|:------|
| dhdhgfgh | dsgsdgdg | dsgsdfg | zxczc |
```

**Comment by roman-r-m** [comment]
Just pushed a new version 1.2.0 that mostly fixes this issue. However the lib I am using seems to have a bug and can't deal with tables having leading/trailing spaces in the 2nd line as in your example. I've raised it on their repo.
For now, changing your example to
```
| A   | B   | C   | D   |
|---|---:|:---:|:---|
| dhdhgfgh | dsgsdgdg | dsgsdfg | zxczc |
```

should give correct result.

**Comment by roman-r-m** [comment]
Fixed in 1.2.1 which should be available shortly.
